rfcs/0003-xcmp-picasso-statemine.md

```diff
@@ -0,0 +1,252 @@
+
```

Comment:
Just curious do we generally have a design process that involves RFCs, or this is just for this proposal for XCM?

Reply:
Acala did proposal (see reference in the end), they have forum for that. So just mimic their approach, so use PRs to dicuss.